eradiate/experiments/_rami4atm.py

```diff
@@ -110,18 +121,23 @@ def _surface_validator(self, attribute, value):
 
     integrator: Integrator = documented(
         attr.ib(
-            factory=VolPathIntegrator,
+            default=None,
             converter=integrator_factory.convert,
-            validator=attr.validators.instance_of(Integrator),
+            validator=attr.validators.optional(attr.validators.instance_of(Integrator)),
         ),
         doc=get_doc(Experiment, attrib="integrator", field="doc"),
         type=get_doc(Experiment, attrib="integrator", field="type"),
-        init_type=get_doc(Experiment, attrib="integrator", field="init_type"),
-        default=":class:`VolPathIntegrator() <.VolPathIntegrator>`",
+        init_type=":class:`.Integrator` or dict or None",
+        default="None",
     )
```

Comment:
The correct semantic in such cases (automatic value without complicated coupling) is to use `AUTO`, a private field and a getter which will return a default if the field is set to `AUTO` instead of using a post-init step. See the `Atmosphere` line for this kind of behaviour.